### DISCUSSION TITLE
Using Orion Library with GPU

### DISCUSSION THREAD

**Discussion by ShayAdaddi** [sintel-dev/Orion#536] [General]
Hi, I would like to know if i can use the proposed library with a GPU to accelerate the training ?
If yes, please suggest what is the relevant changes I need to implement / configure in the notebook code

Thanks !

**Comment by sarahmish** [comment]
Hi @ShayAdaddi! Orion uses available GPUs by default.

You can follow the setup configuration proposed in tensorflow https://www.tensorflow.org/guide/gpu to get started. Most pipelines available in Orion utilize tensorflow under-the-hood.

 Let me know if you have any followup questions!
